test/e2e/apps/rc.go

```diff
@@ -60,11 +60,21 @@ var _ = SIGDescribe("ReplicationController", func() {
 		testReplicationControllerConditionCheck(f)
 	})
 
-	It("should adopt matching pods on creation", func() {
+	/*
+		Release : v1.13
+		Testname: Replication Controller, adopt matching pods
+		Description: Replication Controller with a matching selector, should adopt matching pods by checking the owner's reference
```

Comment:
An ownerless Pod is created, then a Replication Controller (RC) is created whose label selector will match the Pod.  The RC MUST either adopt the Pod or delete and replace it with a new Pod.